girder/utility/filesystem_assetstore_adapter.py

```diff
@@ -332,19 +332,39 @@ def _importDataAsItem(self, name, user, folder, path, files,
             self.importFile(item, os.path.join(path, fname),
                             user, name=fname)
 
+    def _hasOnlyFiles(self, path, files):
+        return all(map(os.path.isfile, (os.path.join(path, fname)
```

Comment:
IMO, just get rid of the map:

    return all(os.path.isfile(os.path.join(path, fname)) for fname in files)